CHANGELOG.md

```diff
@@ -4,6 +4,15 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.66.0
+
+- fix: Defer tracing decision to downstream SDKs when using SDK without performance (#8839)
+- fix(nextjs): Fix `package.json` exports (#8895)
+- fix(sveltekit): Ensure target file exists before applying auto instrumentation (#8881)
+- ref: Use consistent console instrumentation (#8879)
+- ref(browser): Refactor sentry breadcrumb to use hook (#8892)
+- ref(tracing): Add `origin` to spans (#8765)
```

Comment:
```suggestion
- ref(tracing): Add `origin` to spans (#8765)
```

Reply:
I wanted to leave these in because they non-trivially touch our console instrumentation and somebody with issues might find the changelog entry useful (in case there are any issues). 

But I can remove the entries if you feel strongly about it.